blogpost3.html

```diff
@@ -0,0 +1,15 @@
+<html>
+
+	<head>
+	<title> On Carrots </title>
+	<link rel='stylesheet' href='blogpost3.css' />
```

Comment:
I don't think this is actually defined anywhere, is it? You normally don't want different css files for each html file. You want one css file used by multiple html files.